lib/active_merchant/billing/gateways/fusebox.rb

```diff
@@ -141,6 +147,12 @@ def void(money, creditcard, options = {})
         )
       end
 
+      def auth_reversal(money, creditcard, options = {})
```

Comment:
`auth_reversal` is a fusebox-specific method and is not one of the "standard" ActiveMerchant gateway methods.  ActiveMerchant calls this a `void`, so we may want to adhere to that signature (from https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateway.rb#L16)

```rb
void(identification, options = {})
```

In the fusebox case, I guess the `reference` would be the value for `identification`, and since they require the PAN (which is dumb, IMHO), it would need to be passed in via the options hash.